src/imago.js

```diff
@@ -112,17 +147,44 @@
         height: self.elements.figure.clientHeight,
       };
 
-      document.addEventListener('mousemove', crop);
-      document.addEventListener('mouseup', stopCropping);
+      if(e.type == 'touchend') {
+        // Touch Event
+        self.tmp.mouse = {
+          x: e.changedTouches[0].pageX,
+          y: e.changedTouches[0].pageY,
+        };
+
+        document.addEventListener('touchmove', crop);
+        document.addEventListener('touchend', stopCropping);
+
```

Comment:
It works for me. Tested on Safari iPad iOS9 and Chrome iPhone 6 iOS9

https://youtu.be/qKUMkuW-tm4